README.md

```diff
@@ -1,2 +1,30 @@
-# scraped_page
+# ScrapedPage
+
 Write declarative scrapers in Ruby
+
+## Usage
+
+Create a subclass of `ScrapedPage` for each page you wish to scrape.
+
+```ruby
+require 'scraped_page'
+
+class ExamplePage < ScrapedPage
+  field :title do
+    noko.at_css('h1').text
```

Comment:
This makes me think that `noko` is possibly exposing too much of the implementation details. Perhaps we could expose the nokogiri methods in the page class directly?

I've opened https://github.com/everypolitician/scraped_page/issues/2 to track this so lets continue the discussion over there ➡️